telegram/ext/_dispatcher.py

```diff
@@ -631,6 +652,78 @@ def remove_handler(self, handler: Handler, group: int = DEFAULT_GROUP) -> None:
             if not self.handlers[group]:
                 del self.handlers[group]
 
+    def drop_chat_data(self, chat_id: int = None, all_empty_entries: bool = False) -> None:
+        """Used for deleting a key from the :attr:`chat_data`.
+
+        .. versionadded:: 14.0
+
+        Args:
+            chat_id (:obj:`int`, optional): The chat id to delete from the persistence. The entry
+                will be deleted even if it is not empty. Mutually exclusive with
+                :paramref:`all_empty_entries`.
+            all_empty_entries (:obj:`bool`, optional): If :obj:`True`, all empty entries from
+                :attr:`chat_data` will be deleted. Mutually exclusive with :paramref:`chat_id`.
```

Comment:
"empty" doesn't necessarily have a meaning when using a custom type for `chat_data`. Maybe we could use smth like "evaluates to False in boolean context i.e. ``bool(entry)=False``"? 🤔

Reply:
yes. We should also change the param name then. Any suggestions?

Reply:
mh, not easy … maybe `all_falsy`? `Falsy` doesn't seem to be an offical term (couldn't find it on docs.python.org), but it seems to at least be widely used …

Reply:
didn't change the param name since it would probably cause confusion imo